### ISSUE TITLE
J2me Loader on Kai OS?

### ISSUE THREAD

**Issue by AniXL** [nikita36078/J2ME-Loader#1021]
### Quick summary

Not a exactly a feature request.

### Details

Is it possible to port J2me Loader over to Kai OS?

I've seen some emulators that run on browsers like one for Nintendo DS and Kai OS is a web-based OS for feature phones. So the idea is emulating Java games on modern feature phones as a web app, which will be really awesome to see!

**Comment by SergioFLS** [comment]
Not possible easily unfortunately. J2ME-Loader is written in Java with heavy use of Android-specific APIs, making porting to other platforms harder.

**Comment by Thetouchedjoe** [comment]
yeah porting to KaiOS kinda imposible, like you need to resize the screen and removing some much features to be able to support somehow

**Comment by zixing131** [comment]
you can try this：https://github.com/zixing131/myjump
